README.md

```diff
@@ -64,15 +66,17 @@ Jump to [_**example-stacks**_](https://github.com/Stack-Box/StackBox/blob/master
 | **Rails** | ✅ | ❌ | na | na | ❌ | na | ❌ |
 | **Vue** | ✅ | ✅ | na | na | ❌ | na | ❌ |
 | **Angular** | ✅ | ✅ | na | na | ❌ | na | ❌ |
+| **React** | ✅ | ✅ | na | na | ❌ | na | ❌ |
```

Comment:
Let's change this line like below. We make them green as we add connectivity to specific backend endpoints.
| **React** |  ❌ |  ❌ | na | na | ❌ | na | ❌ |

Reply:
OK